### ISSUE TITLE
D2R Ladder 1 Journal

### ISSUE THREAD

**Issue by superarts** [superarts/superarts.github.io#3]
Day 1-3: Classic c-rush sorc fully built, rush done in 1 hour

And made about 2000 fg. People were asking 1000 for crush on day one, but it lowered to 200 fg in 3 days. I did a bunch at 300 fg mostly - guess that's the price people are comfortable with, which is about an `Ist` (280 fg on day 3).

Day 4: Building expansion sorc
- lv 1-15: trist
- lv 15-25: tomb
- lv 25-50: normal baal
- lv 50-60: solo nightmare cow, literally no gear is needed
- lv 60-68: hell baal
- lv 68-72: solo hell cow, starting with 0 gear
  - lv 68: stealth
  - lv 71: monach

Basically with the good old Blizzard, cow level is the best place you can solo XP with literally zero gears (still need level 50 so you can have a larger mana pool at 200 ish), and hopefully you can get some low runes and bases too. I'm pretty unlucky today that I couldn't many bases, and even at this point, I still don't have a 4os sword.

**Comment by superarts** [comment]
With spirit shield and stealth, I have 53 FCR and this makes me able to host i-tele baal runs. Without BO, I have only 500 hp, so I occasionally die during TP. But it's good to find a public game that's not full of leechers, so I made 10 g games and leveled to 77.

My awesome gears:
![image](https://user-images.githubusercontent.com/491257/166176275-c2ddfbd2-9ed8-43a2-bc4f-2b4cec4abbfe.png)
![image](https://user-images.githubusercontent.com/491257/166177718-a4b8dc48-3f20-48aa-ac7a-506e3198bfd9.png)

![image](https://user-images.githubusercontent.com/491257/166177655-964116bf-b7b8-4eef-9163-9c7a89b814e2.png)
![image](https://user-images.githubusercontent.com/491257/166177669-550c162a-a50e-463d-8624-b16fd2744359.png)
![image](https://user-images.githubusercontent.com/491257/166177685-6f98f5db-34b9-4eba-a5a8-d6c5c000367a.png)
![image](https://user-images.githubusercontent.com/491257/166177704-b349f5b5-0094-4b34-b2b8-7493418d58b9.png)

Someone gave me a FCR ring so I could hit 63 FCR (I was using a 13 MF blue ring):
![image](https://user-images.githubusercontent.com/491257/166177065-dc9008ee-9c9b-4dc2-a354-d50a19c35636.png)

And finally we are level 80.
![image](https://user-images.githubusercontent.com/491257/166178947-fcceeaa7-48a5-4cf3-b684-72c2d27f60a6.png)

**Comment by superarts** [comment]
Did normal and nightmare missing quests. Now gonna run some countess cuz she's easy, that my merc can deal with her with these gear:

![image](https://user-images.githubusercontent.com/491257/166183107-99d8323c-9f59-4f70-ae82-11eba7b328e6.png)
![image](https://user-images.githubusercontent.com/491257/166183116-893ee99c-158d-4825-ac3c-9d10360fb086.png)
![image](https://user-images.githubusercontent.com/491257/166183130-b74c8aae-b845-4745-bb95-031cf265c7cb.png)

**Comment by superarts** [comment]
I think I'm done countess for now, built my low key stash (sold lem for 100 fg, ~~will try to sell 2 terror keys~~ sold for 170 fg)

![image](https://user-images.githubusercontent.com/491257/166187038-08011d66-9f52-41a7-bcab-4030629d3dad.png)

**Comment by superarts** [comment]
Did a bunch of andy / duriel / meph. Without high damage insight and leech helm, merc can't stay alive so I switched to focusing on meph. Got a 32/9 magi, sold for 425 fg. It's a nice replacement of my stealth for sure, all res and +1 skill increases offense and defense but they are not game changer, and 5 fcr does nothing at this point. For MF, you just need MF and FCR, especially if we are doing meph block kill (close up static field kill requires higher def, and merc needs to be able to tank damage too).

Good first day of my sorc! I didn't really purchased anything at all, unlike my classic sorc (brought shard for 600, sold for 1200, found another one from andy lol)

**Comment by superarts** [comment]
Brought 6 topaz for 33 fg, 4 os chest for 10 fg, so Stealth out, my random rare ring out:
![image](https://user-images.githubusercontent.com/491257/166226257-8a3cb60a-4baa-4f3e-8e67-3e541f586289.png)

Replaced with random weapon and 10 FCR ring:
![image](https://user-images.githubusercontent.com/491257/166226546-8519c338-cee1-45e1-a74b-2c6e6f11af33.png)
![image](https://user-images.githubusercontent.com/491257/166226581-c8e4f061-af1a-4329-8a2e-88ba8afb1a45.png)

4 topaz chest in, also cut aldur for random mf glove and boots in:
![image](https://user-images.githubusercontent.com/491257/166226315-2279e503-5d2a-4af6-aaee-66dc5323e4ab.png)

Now we have 68 FCR, 226 MF.

**Comment by superarts** [comment]
Got 10ll andy visage, [sold for 350](https://forums.d2jsp.org/topic.php?t=93391350&f=271&p=610305909#p610305909).

Made edge to start gambling. Burned 600k on plated boots, 600k on lv11 dagger and got nothing (you can use edge nor gheed charm due to level restriction). But with IGG, it's actually very easy to gamble for Gull, and that's why it's pretty cheap - 100 fg right now. Doesn't really worth the time to gamble as it would take an hour to burn 10 mil gold.

**Comment by superarts** [comment]
Brought 32 fcr spirit sword for 30, and sold 3os kite shield for 40 before work. (This is the price difference between expansion and classic!)

**Comment by superarts** [comment]
After getting a perfect mara from meph, which I'm still keeping as I can't sell it at 2000, I decide to further building my char. Starting with my merc:
- Tal helm for 40
- Made a non-eth wire fleece Smoke as it's cheaper than treachery or lionheart, because I used my lem to:
- Trade my lem with a non-eth great poleaxe (14 aura 253 ED)

None of these are BIS obviously, but this would allow my merc to at least do something in hell other than dealing with countess (and now he can facetank the full countess encounter).

With the help of Mara, I don't really have a huge problem with my sorc. I replaced chest with Stealth and added FCR rings back to achieve 105 FCR, and replaced MF helm with a random helm:
![image](https://user-images.githubusercontent.com/491257/166399103-422837fd-0eb7-4282-b1f6-5bfb714b2e32.png)

Aldur boot is back with a random pair of laying of hands.

This setup allow me to do some runs like andy / duriel, but it's not really my goal. There's no point to MF without high MF gear anyway. I'm just testing my power level, and with this setup, I did all hell quests like hellforge, anya etc. Not smooth but doable. The REAL goal is to pass trav and CS so that I can do self-crush, hellforge, and socket quest runs.

However, during testing, I found a trang glove, so I can get rid of my FCR rings. I can barely do trav and CS, but the efficiency may be way too low.

**Comment by superarts** [comment]
Sold mara for 2000 fg, now I don't need to worry about my gear to decrease value while I'm working, as I don't have any expensive gear on me now.

I brought a goldwrap for 20 and up'ed it myself, brought an IK helm for 15, and realized that it's a barb only item. With my MF helm, I'm able to achieve 120 MF, which is fair enough for key runs I guess.

I also rushed my mule to hell A4, killed diablo and realized my mule wasn't there. CS takes too long for me anyway, as I'm running MF gear and my res is nearly 0%. Kira helm would be a nice update but guess I'll get it later. I did all 3 hellforges which gave me nothing, and 2 socket quest and sold them for 70 fg each.

With my currencies, I think I can craft some jewels as my rings and amulet are really, really terrible, especially after I sold the perfect Mara. I didn't even realize I was running hell no amulet at all...

And I realized that I don't have IGG. Gonna buy some but it's too late now, just sold a terror key I got today for 58 fg and that's it.

**Comment by superarts** [comment]
Spent 115 fg
- 2 mil IGG: 20
- Tal helm + magefist: 55
- Tal belt 10 mf: 10
- Kira: 30

Crafted some amulets and rings and they are totally garbage, and it turned out that 2 piece tal set is a pretty bad combination so I decide not to use them for now. So we have combat 105 FCR, 50+ res setup:
![image](https://user-images.githubusercontent.com/491257/166588017-01216dfd-c798-411f-b550-f6ef7ce7808c.png)

And 257 MF, 63 FCR setup:
![image](https://user-images.githubusercontent.com/491257/166589639-0b2ced1d-804c-40d9-aae0-a84632050755.png)

Combat setup is only good for key runs, and MF setup is only good for Meph as the main problem is to keep merc alive. I'll do a couple of runs and hope I can make a treachery.

(I have random goals here and there, my fun comes from the balance of playing it efficiently, and looting by myself. Most players do have dedicated goal like SSF to BIS lightning sorc, fastest fg etc. but it's not me.)

**Comment by superarts** [comment]
Sold Mal for 80 but I made a HUGE mistake that I missed the diablo clone walk event on Asia server at 8:00 PM EST, due to some personal issues. I became really frustrated and did a bunch of Baal runs, and leveled from 81 to 84.
![image](https://user-images.githubusercontent.com/491257/166622303-4ebee8e0-f7f6-416b-aa68-872a136ce921.png)

Later on, I brought a lem for 50 fg, made a treachery, and used my socket quest to make a thresher obedience. However, this combination still doesn't really work, as I can't kill Andy or Meph before they kill my merc. Maybe fortitude is the only answer to master hell? Or CTA will be the next game changer?

Anyway, I finished my first mule's hell socket quest and sold for 70 fg. At this point, it's hard to make further changes without more investment, and I think I'll back to rush again later. Still grumpy about losing 2400 fg today (1200 unid anni)... That's how much information is worth!

**Comment by superarts** [comment]
I finally figured out [multi-boxing in 2.4](https://www.reddit.com/r/Diablo/comments/ui7z5b/multiboxing_d2r_24_playing_now/) again, and rushed 3 mules of mine (bought from fg and BN balance). The classic part is easy, trav is not as bad as I expected but it's still far from efficient, and CS became a total nightmare. I mean I was having fun fight myself, but if I were rushing others, they would definitely became impatient.

From 6 hellforge, I got pul + um + mal and sold for 150 (lem is about 25 now btw). Out of 12 socket quests, I sold 4 so far for 280, so got totally 430 fg from this point. I'm surprised to see lem dropped from 100 to 25 in 2 days but socket quest stays the same. It has to drop soon.

**Comment by superarts** [comment]
Sold a bunch of socket quests. Rushed my own hell-ready mule and helped 2 guys for their forges doing a2-a5 (my mule did a3 organs, and one guys was asking for a2 - and I returned his Io from forge). Got Gul from another guy and pul from my mule, sold for 235 fg. I'll probably stop recording socket quests and runes, as Gul is the best you can get from forge and it can be sold instantly at 200, so you can't really get luckier than that. Socket quests are a better income source than runes now (probably we'll see 60 fg or even 50 tomorrow).

I'm still using Stealth, and I did a final CS run to level to 85, and got Que-Hegan's Wisdom. This is why I don't buy GG gears, as I can still upgrade from my own MF which is fun.

Crafted a bunch of jewelries and got 2 replaces at least, amulet is fine but ring is terrible:
![image](https://user-images.githubusercontent.com/491257/166868170-783426a6-cde6-4298-b37b-a050336993e0.png)
![image](https://user-images.githubusercontent.com/491257/166868221-00f4841c-7f42-4a07-a4f2-07fdfae82db6.png)

So yea my gear is still pretty bad.

**Comment by superarts** [comment]
Got 2 hate keys but it's too late to sell. I'm getting tired of sorc now, maybe I'll level a paladin before POE 3.18 comes out (tomorrow is the reveal day, my favorite 3.17 build is skeleton mage -> cremation, pretty sure both will be nerfed)

3.16 I played toxic rain and it was nerfed to the ground in 3.17 (yet people on reddit were claiming it was "fine", those stupid downvoting bastards on r/PathOfExile are the worst), my 3.17 league start build seismic trap was nerfed too... Path of Nerfing indeed.

**Comment by superarts** [comment]
Leveled my hammerdin to 73 today, got some great CS carrier:
![image](https://user-images.githubusercontent.com/491257/167051528-108d7d61-54d5-46a2-b4b6-5970bc4c7fce.png)

FoH seems to be quite nice when fully geared as you don't need to suffer hammer's painful ballista anymore.

**Comment by superarts** [comment]
Did a bunch of baal last night and got my paladin to 84, and I'm building it today.

- Defense oriented: 125 FCR, 90 all res, 75 block rate (so only 5k damage with garbage gear below, no anni no torch)
- Doing all the quests
- Gear up & flip if possible
  - ptorch 18/12: 800 -> 950
  - cta 3/1/3: 500 -> 650
  - cta 6/1/3: 525 -> 600

Still with a lot of temporary gears like:
![image](https://user-images.githubusercontent.com/491257/167174158-34d9f5ba-153e-416d-9c33-74c3bbfb7410.png)
![image](https://user-images.githubusercontent.com/491257/167174186-b35835f9-4ecd-470b-80cc-316a4aeabce1.png)
![image](https://user-images.githubusercontent.com/491257/167174205-73acbe0c-7279-41f6-aa24-7272ebcf4147.png)

FoH might be better in terms of cleaning and I'm going to test out later.

**Comment by superarts** [comment]
Not a lot going on in the weekend. Here are the deals I made:
- Gul: 115
- Pul: 20
- T key: 20
- Socket quest: 50
- Crush: 150

Ber is around 2000 so we are at a "mid ladder" point. Last time when Ber was around 1000, 2.4 was already announced. Now on standard Ber is 400 (for a "dead" realm). I'm surprised to see people progressing so fast in the first ladder, which is not surprising though as there must be a lot of bots working day and night. Due to the Diablo Clone change, tons of cdkeys are out there for 0 purpose other than botting :D

I've been running crushes and MF with my sorc. With anni and storch (brought from 850, my biggest spending so far), it's pretty safe to run cuntess -> andy -> ancient tunnel -> summoner -> duriel -> kurast chests -> milf -> maybe h key. Pindleskin doesn't benefit from 3 ppl, and shenk is ice/fire immune and is tankier than countess, so A1 to A3 makes more sense with my 2 parking chars.

Without Enigma, I just can't get paladin to work. FoH turns out to be a slightly better version than hammer, due to the fact that you don't have to dive into mobs, but with TP on off-hand, it's hard to re-position merc and without fortitude, merc just constantly dies (and with high damage, MF sorc can keep merc alive much better at boss fights). So next ladder perhaps I won't spend time leveling paladin anymore, and will just hire a uber killer to do diablo clone like I did last time.

**Comment by superarts** [comment]
Sold 3 unid anni for 898, so it's 1000 -> 500 -> 300 per. Made a mistake by creating room without password, and made another mistake by trying to create a new room right upon DC was spawning - so 300 less fg. Huge disaster for me:
1. Missed 4000 fg OMG
2. Could have sold 3 anni for 1500 but only for 500 because I identified them
3. Missed another 300 (this time)

So I missed 5800 fg from anni, which is a bit less than all the fg I made during Ladder 1. How stupid.

Anyway,this time, I did it with my own FoH paladin. You only need:
1. Fem aura 1 point for IAS
2. Smite 1 point, switch to a high damage shield to fastern the kill as skill doesn't matter too much
3. Dracul + Golbin toe

At this point, I have 10 socket quests in back (500 fg), a stash of gems and a handful of Mal. I think I'm mostly done this ladder.

PS: also missed a DC walk on EU this morning.
